### ISSUE TITLE
iOS 14 orange microphone privacy indicator on as long as engine is running

### ISSUE THREAD

**Issue by ClassicalDude** [AudioKit/AudioKit#2273]
The stable configuration of my project was built, according to Aure's advice, without calling AudioKit.stop() (now AKManager.stop()). Instead, I just deactivate individual nodes when I was done using them. If I try to call AKManager.stop() after deactivating all nodes, restarting the engine is buggy in the best case scenario, and crashes the app in the worst case scenario.

The problem: as of iOS and iPadOS 14, there is an orange indicator at the top bar whenever an app accesses the microphone. This indicator only turns off when calling AKManager.stop(). As a result, I am afraid my users will think the app is "spying" on them, because the indicator is on when they are doing things that are not microphone-related.

Is there a way to create a stable restarting scheme for the AudioKit engine, or find a workaround that turns off the privacy indicator?

**Comment by aure** [comment]
Do you know if this behavior persists on v5-develop? If you can make a simple app that demonstrates the problem, I can have a look. I think the microphone usage indicator is up as long as your signal chain uses the microphone in any way.

**Comment by ClassicalDude** [comment]
@aure, I don't know about v5-develop. I tried creating a bare-bones app with v5, but by now the difference from v4 is so great, that I couldn't even find how to connect a microphone.

I created a project for you, using the 4.11.1 XCFramework. The project confirms my observation. Please download it [here](https://drive.google.com/file/d/1uwqZhFEdEcrviPoFPRFN61KLrT6gCHWA/view?usp=sharing).

**Comment by ClassicalDude** [comment]
In the meantime, I think I found an awkward workaround. I use a `if !AKManager.engine.isRunning` block to define most of my nodes and their connections. It seems that redoing the definitions and connections after restarting the engine causes the problems. I added a global Bool variable named `audioKitStartedOnce` and set it to `true`after stopping the engine, and then used it to avoid redefining and reconnecting the nodes when restarting the engine. Things seem to be working and stable now, and the orange indicator goes off when the engine is off.
